.github/workflows/CI.yml

```diff
@@ -141,7 +140,10 @@ jobs:
 
       - name: Run mypy
         # subtle differences in Python/mypy versions, just keep it passing on one
-        if: ${{ matrix.rdkit == true && matrix.openeye && matrix.python-version == 3.10 }}
+        # prefer having htis back to rdkit == true, but builds often have bugs in stubs and
+        # there are other bugs/subtleties (in the source code and builds) that reduce the number
+        # of available builds
+        if: ${{ matrix.rdkit == false && matrix.openeye && matrix.python-version == 3.10 }}
```

Comment:
Like it says - a lot of RDKit conda packages have bugs in their stubs that prevent type-checking, and type-checking only in the `rdkit == false` environments is better than nothing